ruby/basic_ruby/predicate_enumerable_methods.md

```diff
@@ -115,39 +120,39 @@ numbers.any? { |number| number < 20 }
 ### The all? method
 The `all?` method is also fairly intuitive. It only returns `true` if *all* the elements in your array or hash match the condition you set within the block; otherwise, it will return `false`.
 
-Let's say that we want to check whether all the words in our list are more than 6 characters long. First,let's see how we could achieve this using `#each`:
+Let's say that we want to check whether all the words in our list are more than 3 characters or 6 characters long. First,let's see how we could achieve this using `#each`:
```

Comment:
Nit: I don't mind including break if the example are still easy to follow, but I don't think we should use break with the examples for `#all?`. The reverse logic makes them harder to reason about and takes away from the main point were trying to get across. 

Our main goal isn't performance with the examples here, its showing learners this is how you _might_ achieve a similar thing if enumerables didn't exist and demonstrates how much code extra they save you from writing.

Reply:
Ok, agreed. I will revert back changes for ```all?```

Reply:
@KevinMulhern I have updated the PR